packages/api/config/database.js

```diff
@@ -0,0 +1,13 @@
+module.exports = ({ env }) => ({
+  connection: {
+    client: 'postgres',
+    connection: {
+      host: env('DATABASE_HOST', 'localhost'),
+      port: env.int('DATABASE_PORT', 5432),
+      database: env('DATABASE_NAME', 'wheblog'),
+      user: env('DATABASE_USERNAME', 'wheblog'),
+      password: env('DATABASE_PASSWORD', 'wheblog'),
```

Comment:
this is for testing purposes, then it can be replaced by ENV variables in docker